src/dashboard/post_distribute/post_distribute.py

```diff
@@ -0,0 +1,51 @@
+"""Handler for validating the payload contents and submitting to queue"""
+
+import json
+import os
+
+import boto3
+import requests
+
+from shared import functions
+
+sns_client = boto3.client("sns", os.environ.get("AWS_REGION"))
+valid_chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-._~:/?#[]@!$&'()*+,"
+
+
+def validate_github_url(config):
```

Comment:
nit: minor naming bump - this not only validates but also does the request - which might be surprising if you just saw the function name.

Reply:
to my mind, 'validate' in this context is 'validate the url is actually a git repository'. Open to tweaks.

Reply:
Oh I guess I thought this was doing more than just pinging the URLs and actually doing downloads. I think I  just misread it.